doc/queue.md

```diff
@@ -0,0 +1,167 @@
+# Queue
+
+## Motivations
+
+In practical applications, multiple tenants often share a cluster, and resource isolation between tenants is required. We need to support Queue Unit under multiple queues.
+
+### Goals
+
+- Support multiple queue queuing
+
+## Proposal
+
+A "CRD" is needed to interact with the kube-queue. The CRD kind name is subject to change. We use "Queue" in the proposal. The CRD defines the information related to multiple queue queuing. "Queue" is namespace scoped.
+
+```yaml
+apiVersion: scheduling.x-k8s.io/v1alpha1
+kind: Queue
+metadata:
+  name: queue
+  namespace: queue
+spec:
+  queuePolicy: Priority # queuing strategy
+  priority: 100
+  priorityClassName: high-priority
+```
+
+The API and Status are described below:
+
+```go
+type Queue struct {
+metav1.TypeMeta   `json:",inline"`
+metav1.ObjectMeta `json:"metadata,omitempty" protobuf:"bytes,1,name=metadata"`
+
+Spec   QueueSpec   `json:"spec,omitempty" protobuf:"bytes,2,name=spec"`
+Status QueueStatus `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
+}
+
+type QueueSpec struct {
+QueuePolicy         QueuePolicy    `json:"queuePolicy,omitempty" protobuf:"bytes,1,opt,name=queuePolicy`
+Priority            *int32         `json:"priority,omitempty" protobuf:"varint,2,opt,name=priority"`
+PriorityClassName   string         `json:"priorityClassName,omitempty" protobuf:"bytes,3,opt,name=priorityClassName"`
+}
+
+type QueueStatus struct {
+// TODO
+}
+
+type QueuePolicy string
+
+const (
+QueuePolicyFIFO       QueuePolicy = "FIFO"
+QueuePolicyPriority   QueuePolicy = "Priority"
+)
+```
+
+### Lifecycle of CRD
+
+#### Create CRD
+
+#### Update CRD
+
+### Delete CRD
+
+## Use Case
+### 1. Create Queue and ResourceQuota for two namespace
```

Comment:
add  command to create queue ？
like `$ kubectl create -f examples/multiqueues/queue1.yaml`?